src/utils/common.ts

```diff
@@ -314,14 +314,48 @@ export const trustarcScriptSetup = () => {
   document.body.appendChild(trustarcScript);
 };
 
-export const loadFedModules = () =>
-  axios.get(`${window.location.origin}${isBeta() ? '/beta' : ''}/config/chrome/fed-modules.json?ts=${Date.now()}`, {
-    headers: {
-      'Cache-Control': 'no-cache',
-      Pragma: 'no-cache',
-      Expires: '0',
+const CHROME_SERVICE_BASE = '/api/chrome-service/v1';
+export const chromeServiceStaticPathname = {
+  beta: {
+    stage: '/static/beta/stage',
+    prod: '/static/beta/prod',
+  },
+  stable: {
+    stage: '/static/stable/stage',
+    prod: '/static/stable/prod',
+  },
+};
+
+export function getChromeStaticPathname(type: 'modules' | 'navigation') {
+  const stableEnv = isBeta() ? 'beta' : 'stable';
+  const prodEnv = isProd() ? 'prod' : 'stage';
+  return `${CHROME_SERVICE_BASE}${chromeServiceStaticPathname[stableEnv][prodEnv]}/${type}`;
+}
+
+export const loadFedModules = async () => {
+  const commnHeaders = {
+    'Cache-Control': 'no-cache',
+    Pragma: 'no-cache',
+    Expires: '0',
+  };
+  // we need to fetch both old and new to get the chrome config from FEO
+  const [FEOModules, staticModules] = await Promise.all([
+    axios.get(`${window.location.origin}${isBeta() ? '/beta' : ''}/config/chrome/fed-modules.json?ts=${Date.now()}`, {
```

Comment:
We are not getting the fed-modules config from FEO. I don't think we need this thing.

Reply:
I think we still need this for EE right? For the SSO URL and stuff. And also console.dev

Reply:
Aaah, yes! For the ephemeral environment we'll need this. For the console.dev not anymore.